src/main.lua

```diff
@@ -418,8 +418,47 @@ local function addPrimaryNameRequestData(ioEvent, primaryNameResult)
 	addPrimaryNameCounts(ioEvent)
 end
 
+local function assertTableBytesLowerThan(tbl, max_bytes)
+	local seen = {}
+	local current_size = 0
+
+	local function sizeof(val)
+		if current_size > max_bytes then
```

Comment:
Shouldn't this be at the end of the function? If it were I think you could remove it from the table section.